docs/getting_started.md

```diff
@@ -212,6 +212,15 @@ spec:
       secretNamespace: kube-system
     network:
       name: vm-network-1
+    providerConfig:
+      cloud:
+        controllerImage: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.0.0
```

Comment:
Hi @andrewsykim,

In the `cluster.yaml` these values are quoted, but they aren't here. Just a small nit.

Reply:
I think this is true of all values though, something in the generator script (I think kustomize) removes the quotes on string values